### PR TITLE
Add portfolio's cards on the 'works' section and 'about me' section content

### PR DESCRIPTION
**Pull request by mrjuda**
Basic html + css structure to the portfolio + about sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,6 +79,191 @@ <h2>LET'S CONNECT</h2>
...
+          </ul>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required</p>
+          <ul class="keyword">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
```
Comment (Donard97) [inline index.html]: Kindly change this div to a button or a link (a tag). Keep coding 💪

File: index.html [hunk 2]
```diff
@@ -79,6 +79,191 @@ <h2>LET'S CONNECT</h2>
...
```
Comment (Donard97) [inline index.html]: Kindly change this div to a button or a link (a tag). Keep coding 💪

File: index.html [hunk 3]
```diff
@@ -79,6 +79,191 @@ <h2>LET'S CONNECT</h2>
...
```
Comment (Donard97) [inline index.html]: Kindly change this div to a button or a link (a tag). Keep coding 💪

File: index.html [hunk 4]
```diff
@@ -79,6 +79,191 @@ <h2>LET'S CONNECT</h2>
...
```
Comment (Donard97) [inline index.html]: Kindly change this div to a button or a link (a tag). Keep coding 💪

File: index.html [hunk 5]
```diff
@@ -79,6 +79,191 @@ <h2>LET'S CONNECT</h2>
...
+          </a></li>
+          <li><a href="#">
+            <img src="icons/github.png" alt="Github logo">
+          </a></li>
+          <li><a href="#">
+            <img src="icons/instagram.png" alt="Instagram logo">
+          </a></li>
+        </ul>
+        <div class="cta-button">Get my resume</div>
+      </div>
```
Comment (Donard97) [inline index.html:215]: Kindly change this div to a button or a link (a tag). Keep coding 💪

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @mrjuda , ✋ 

You did a tremendous job in this project! 💯 💯 💯 
There are some issues that you still need to work on to go to the next project but you are almost there! 👏 

giphy (3)

## Highlights

- Git/GitHub flow best practices have been followed ✅ 
- There are no linter errors ✅ 

## Required Changes ♻️

- [x] Please change the description of PR and do it more descriptive. You have to tell us exactly what you did on this branch.
- [x] The commit messages should be meaningful. Try to correct this behavior in the future. Check out this [link ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)to learn how to document your project professionally.


**_Check the comments under the review._**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 6]
```diff
@@ -6,7 +6,7 @@
 - CSS
 
 ## Live Demo
-- Live version: NO LIVE VERSION YET
+- Live version: [mrjuda.github.io/judaport](https://mrjuda.github.io/judaport)
```
Comment (MrBrN197) [inline README.md:9]: - [x] Hi you've made I change here however, I notice it is the only change you've made. please Look at the previous reviewer's comments and make all his suggested improvements. 
- [x] Have a look at the inline comments and kindly make the necessary changes.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @mrjuda  👋 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mrjuda) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (mrjuda) [reply index.html]: done

File: README.md (hunk 6 above)
Comment (mrjuda) [reply README.md:9]: done it

File: index.html (hunk 3 above)
Comment (mrjuda) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (mrjuda) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (mrjuda) [reply index.html:215]: done

**Review by Wes-Isaac — APPROVED** [review]
Hi @mrjuda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**